src/index.ts

```diff
@@ -16,6 +16,7 @@ import {
 } from "./componentprocessor";
 import { Parser } from "./parser";
 import { Interpreter, ExecutionOptions, defaultExecutionOptions } from "./interpreter";
+export { resetTestData } from "./extensions";
```

Comment:
I'm not sure that this needs to be exposed from the root of the module.  If it does need to be exposed, I'd like for us to find (or add) a new property to put it on instead of it being a sibling of `brs.lexer`, `brs.parser`, `brs.types`, etc.  Those are all very low-level concepts, and `resetTestData` doesn't match those 😃